src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -802,6 +802,50 @@ public String toString() {
     }
   };
 
+  public static final Builder<StreamAutoClaim> STREAM_AUTO_CLAIM = new Builder<StreamAutoClaim>() {
+    @Override
+    @SuppressWarnings("unchecked")
+    public StreamAutoClaim build(Object data) {
+      if (null == data) {
+        return null;
+      }
+
+      List<ArrayList<Object>> objectList = (List<ArrayList<Object>>) data;
+      if (objectList.isEmpty()) {
+        return null;
+      }
+
+      StreamEntryID nextStartEntryID = STREAM_ENTRY_ID.build(objectList.get(0));
+      objectList.remove(0);
```

Comment:
I think we use `index of list` from `1` directly would be better and no need to call `remove`.

Reply:
Yes. Removed the remove(0) statement and used subList from index 1